cmd/node/config/gasSchedules/gasScheduleV2.toml

```diff
@@ -68,7 +68,7 @@
     ExecuteOnDestContext = 160000
     DelegateExecution    = 160000
     AsyncCallStep        = 200000
-    AsyncCallbackGasLock = 20000000
+    AsyncCallbackGasLock = 2000000
```

Comment:
10x cheaper?

Reply:
@sasurobert ?

Reply:
yes. it was way too much. we have an additional dynamic gas lock depending on the size of the contract. which gets accumulated with this.